app/views/works/_work_header_navigation.html.erb

```diff
@@ -51,10 +51,15 @@
       <%= link_to ts("Cancel Bookmark"), "#", :class => "hidden bookmark_form_placement_close" %>
     </li>
 
-    <% unless current_user.is_author_of?(@work) || current_user.try(:preference).try(:history_enabled) == false || marked_for_later?(@work) %>
-      <li class="mark"><%= marktoread_link(@work) %></li>
+    <% unless current_user.is_author_of?(@work) || current_user.try(:preference).try(:history_enabled) == false %>
+     <li class="mark"> 
```

Comment:
I think the indenting is off here -- this looks like one space instead of two. Also, line 57 will need to be `mark_as_read_link(@work)` to go with the change requested in [my comment above ](https://github.com/otwcode/otwarchive/pull/2481/files#r66373553)